specs/introduction.md

```diff
@@ -0,0 +1,127 @@
+# Optimistic Ethereum Introduction
+
+<!-- START doctoc generated TOC please keep comment here to allow auto update -->
+<!-- DON'T EDIT THIS SECTION, INSTEAD RE-RUN doctoc TO UPDATE -->
+**Table of Contents**
+
+- [Foundations](#foundations)
+  - [What is Ethereum scalability?](#what-is-ethereum-scalability)
+  - [What is Optimistic Rollup?](#what-is-optimistic-rollup)
+  - [What is EVM Equivalence?](#what-is-evm-equivalence)
+  - [🎶 All together now 🎶](#-all-together-now-)
+- [Protocol Guarantees](#protocol-guarantees)
+- [Network Participants](#network-participants)
+  - [Users](#users)
+  - [Sequencers](#sequencers)
+  - [Verifiers](#verifiers)
+- [Key Interaction Diagrams](#key-interaction-diagrams)
+  - [Depositing and Sending Transactions](#depositing-and-sending-transactions)
+  - [Withdrawing](#withdrawing)
+- [Next Steps](#next-steps)
+
+<!-- END doctoc generated TOC please keep comment here to allow auto update -->
+
+Optimistic Ethereum is an _EVM equivalent_, _optimistic rollup_ protocol designed to _scale Ethereum_ while remaining maximally compatible with existing Ethereum infrastructure. This document provides an overview of the protocol to provide context for the rest of the specification.
+
+## Foundations
+
+### What is Ethereum scalability?
+
+Scaling Ethereum means increasing the number of useful transactions the Ethereum network can process. Ethereum's limited resources, specifically bandwidth, computation, and storage, constrain the number of transactions which can be processed on the network. Of the three resources, computation and storage are currently the most significant bottlenecks. These bottlenecks limit the supply of transactions, leading to extremely high fees. Scaling Ethereum and reducing fees can be achieved by increasing the supply of these limited resources.
```

Comment:
Scaling is increasing the supply of gas while holding the hardware constraints constant. Another way to put that is that scalability increases the throughput of the system while not increasing the minimum hardware requirement.

Reply:
> Scaling Ethereum and reducing fees can be achieved by increasing the supply of these limited resources.

Scaling ethereum and reducing fees can be achieved by better utilizing bandwidth, computation and storage. Increasing the supply of those 3 isn't really increasing the scalability of the protocol itself